nibabel/freesurfer/tests/test_io.py

```diff
@@ -209,6 +209,101 @@ def test_annot():
         assert_equal(names, names2)
 
 
+def test_read_write_annot():
+
```

Comment:
I personally prefer to minimize blank lines, and avoid them at the top of functions.